src/controllers/bookAccomodation.js

```diff
@@ -1,49 +1,107 @@
-import { Accomodation, Room } from '../database/models';
-
-
+import { Accomodation, Room, Bookings, User } from '../database/models';
 
 export default {
-    bookAccomodation: async (req, res) => {
-        const { id, firstName, lastName, phoneNumber, email } = req.user
-        const { roomId, accommodationId, checkInDate, checkOutDate } = req.body
-
-
-        const accomodation = await Accomodation.findOne({
-            where: { id: accommodationId },
-            include: [
-                {
-                    model: Room,
-                    as: 'rooms'
-                }
-            ]
-        });
-        if (!accomodation) {
-            return res.status(404).json({ message: res.__('No accomodation found') });
-        }
+  getBookings: async (req, res) => {
+    const bookings = await Bookings.findAll({
+      include: [
+        { model: Room, include: [{ model: Accomodation }] },
+        { model: User }
+      ]
+    });
```

Comment:
May you add functionality for only admins to get all bookings, I think it wouldn't be good for everyone to get all this info